packages/api-client/src/team/feature/FeatureAPI.ts

```diff
@@ -66,9 +66,6 @@ export class FeatureAPI {
     return response.data;
   }
 
-  /**
-   * @deprecated Use `getAllFeatures()` instead.
```

Comment:
Actually, shouldn't it be the other one that is deprecated?

Reply:
this one is actually not deprecated as you say, however the regular GET feature-configs/ endpoint without a feature attached is also not deprecated